Python/samples/simple_console/requirements.txt

```diff
@@ -0,0 +1 @@
+../../libraries/recognizers-number/
```

Comment:
Is this really just a txt file? Just curious.

Reply:
Yes, this file contains a list of items to be installed using `pip install -r requirements.txt`. [(more information)](https://pip.pypa.io/en/stable/user_guide/#requirements-files)